api/python/ai/chronon/staging_query.py

```
@@ -22,6 +22,22 @@ class TableDependency:
     additional_partitions: Optional[List[str]] = None
     offset: Optional[int] = None
 
+    def to_thrift(self):
+        offset_window = common.Window(length = self.offset, timeUnit= common.TimeUnit.DAYS) if self.offset else None
```

Flake8 (7.2.0)

[error] unexpected spaces around keyword / parameter equals

(E251)